app/controllers/integrated/remove_household_member_controller.rb

```diff
@@ -1,6 +1,14 @@
 module Integrated
   class RemoveHouseholdMemberController < RemoveMemberController
     def update_models
+      if member.is_spouse?
```

Comment:
just pointing out something I've been trying—updating models and reconciling the sources of truth wherever we can in the relevant controller, and keeping updates out of the skip.

Reply:
Yup, I agree with this!

Reply:
sweet